src/App.css

```diff
@@ -1,42 +1,7 @@
-* {
-  box-sizing: border-box;
-  padding: 0;
-  margin: 0;
-  scroll-behavior: smooth;
-}
-
-
-
-@media screen and (min-width: 640px) {
-  .container {
-    max-width: 640px;
-  }
-}
-@media screen and (min-width: 768px) {
-  .container {
-      max-width: 768px;
-  }
-  }
-  @media screen and (min-width: 1024px) {
 .container {
-    max-width: 1024px;
+  width: 100%;
 }
-}
-@media screen and (min-width: 1280px) {
-  .container {
-      max-width: 1280px;
-  }
-  }
 
-  @media screen and (min-width: 1536px) {
-    .container {
-      max-width: 1536px;
-  }
-  }
-  
-.container {
-    width: 100%;
-}
```

Comment:
Why these styles are removed?

Reply:
Sir they are there I just moved their location below the line of .container { width:100% }